Moggles/ClientApp/src/featureToggle/EditFeatureToggle.vue

```diff
@@ -179,7 +182,10 @@
                     featureToggleName: this.rowToEdit.toggleName,
                     isPermanent: this.rowToEdit.isPermanent,
                     statuses: statuses,
-                    reasonToChange: !this.stringIsNullOrEmpty(this.reasonToChange) ? this.reasonToChange : null
+                    reasonToChange: {
+                        description: !this.stringIsNullOrEmpty(this.reasonToChange) ? this.reasonToChange : null,
```

Comment:
If reason to change is not provided the whole reasonToChange object should be set to null. Make sure you also modify the controller